### DISCUSSION TITLE
Error while compiling

### DISCUSSION THREAD

**Discussion by utya** [arendst/Tasmota#18881] [Q&A]
Hi. after

#define USE_TOUCH_BUTTONS	

got:

Linking .pio/build/tasmota32/firmware.elf
/root/.platformio/packages/toolchain-xtensa-esp32/bin/../lib/gcc/xtensa-esp32-elf/8.4.0/../../../../xtensa-esp32-elf/bin/ld: /tmp/ccPB0Q2P.ltrans19.ltrans.o:(.literal+0x15c): undefined reference to `buttons'
collect2: error: ld returned 1 exit status
*** [.pio/build/tasmota32/firmware.elf] Error 1

**Comment by Jason2866** [comment]
WiP https://github.com/arendst/Tasmota/pull/18833
